business_requirement_deliverable/models/business.py

```diff
@@ -78,8 +78,6 @@ def product_id_change(self):
             uom_id = product.uom_id.id
         if product.description_sale:
             description += '\n' + product.description_sale
-        if not self.name:
```

Comment:
@rubencabrera As you know the code for onchange on the resource is already merged in #132 
and the test cases to support it is already added in Open PR #139 check this commit
https://github.com/OCA/business-requirement/pull/139/commits/2a02aedc74743772bff01b9e178b4408f01eae1f

So the travis is already fixed in the above mentioned PR.

So IMO you just need to revert your last commit and that will do all.

cc @elicoidal @victormartinelicocorp